### ISSUE TITLE
enhancement request

### ISSUE THREAD

**Issue by mortoza** [marcushellberg/pivottable#4]
Hi
This looks very good add-on. Can I use data from a database table instead of CVS?

**Comment by marcushellberg** [comment]
Yes, I should add support for adding data as arbitrary arrays, this way you have more flexibility in where to pull the data from. If you have time and interest, I'm open to pull requests.

**Comment by mortoza** [comment]
sure. Let me know how I can help.
